user-org-service/app/utils/validator/schema/SchemaFactory.java

```diff
@@ -0,0 +1,39 @@
+package utils.validator.schema;
+
+public class SchemaFactory {
+
+    static volatile SchemaManager INSTANCE;
+
+
+    private SchemaFactory() {
+        throw new AssertionError();
```

Comment:
why SchemaFactory is throwing AssertionError?

Reply:
to make it reflection proof

Reply:
Not needed.